modules/apps/asset/asset-info-api/src/main/java/com/liferay/asset/info/item/provider/AssetEntryInfoItemFieldSetProvider.java

```diff
@@ -30,6 +30,11 @@
 @ProviderType
 public interface AssetEntryInfoItemFieldSetProvider {
 
+	public InfoFieldSet getInfoFieldSet(AssetEntry assetEntry);
+
+	public InfoFieldSet getInfoFieldSet(
+		long scopeGroupId, String itemClassName);
```

Comment:
I think you need to make the parameters in alphabetical order. Also, I believe that semantically it also makes sense to have scopeGroupId second since it's optional.